### PR TITLE
Fixed: Incorrect behavior on IOS for call driveApi.files.getBinary(fileId)

### PR DESCRIPTION
**Pull request by Voznov**

### CODE REVIEW

**Comment by Voznov** [comment]
Fixed #86

File: src/api/aux/utils.ts [hunk 1]
```diff
@@ -14,7 +14,10 @@ export function blobToByteArray(blob: Blob): Promise<Uint8Array | null> {
       }
 
       if (typeof reader.result === 'string') {
-        const b64 = reader.result.split('data:application/octet-stream;base64,')[1]
+        const b64 = reader.result.split('base64,')[1]
+        if (!b64) {
```
Comment (RobinBobin) [inline src/api/aux/utils.ts:18]: Could I kindly ask you to add an empty line before this line (`if (!b64) {`), to make the code slightly more readable, please?

File: src/api/aux/utils.ts [hunk 2]
```diff
@@ -14,7 +14,10 @@ export function blobToByteArray(blob: Blob): Promise<Uint8Array | null> {
...
+          reject({ reader })
+        }
 
         resolve(toByteArray(b64))
```
Comment (RobinBobin) [inline src/api/aux/utils.ts:22]: `resolve(toByteArray(b64))` would be invoked even if `reject({ reader })` was called, if I'm not mistaken. Do you agree?

**Review by RobinBobin — COMMENTED** [review]
Could you review my comments, please?

File: src/api/aux/utils.ts (hunk 2 above)
Comment (Voznov) [reply src/api/aux/utils.ts:22]: It's not true. Reject & Resolve stops a call

**Review by RobinBobin — APPROVED** [review]

**Comment by RobinBobin** [comment]
@Voznov , thanks!

**Comment by XantreGodlike** [comment]
Guys I think split may cause incorrect behaviour if 'base64,' in text

**Comment by XantreGodlike** [comment]
`replace(/^.+?base64\,/, '')`

**Pull request merged**